### ISSUE TITLE
A single audio file for the whole storyboard ?

### ISSUE THREAD

**Issue by gordielachance** [wonderunit/storyboarder#1230]
Hi!  I see that it is possible to attach an audio file for each board.
This is nice but quite a difficult workflow for little projets.

What I would like is a way to add a single audio file for the whole storyboard; would it be possible ?
Maybe we should have an audio timeline in addition of the boards timeline...
What do u think ?

Thanks !

**Comment by redemprez** [comment]
I must absolutely agree that it would be great solution, especially for storyboarding music videos.

**Comment by gordielachance** [comment]
timeline audio VS boards audio :)
And both could play together.
EG you could have a main track for the timeline, and FX per shots.

**Comment by tajinder3d** [comment]
Yes i need this feature too. Will be very useful for making storyboard for songs :) Please make this possible.

**Comment by GeekyStudios** [comment]
Would be fantastic to have an audio track to work with, especially with timing for some sound cues.
